src/include/raft_net.h

```diff
@@ -289,6 +289,12 @@ raft_client_rpc_msg_size_is_valid(enum raft_instance_store_type store_type,
 
 #define RAFT_NET_WR_SUPP_MAX 1024 // arbitrary limit..
 
+
+enum raft_net_wr_supp_op_flag
```

Comment:
You can shorted to `raft_net_wr_supp_op`.  In my opinion a 'flag' is a single bit value which can be combined with other flags.